app/src/main/java/com/android/streetworkapp/model/park/ParkRepositoryFirestore.kt

```diff
@@ -142,7 +142,7 @@ class ParkRepositoryFirestore(private val db: FirebaseFirestore) : ParkRepositor
    */
   override suspend fun addRating(pid: String, rating: Int) {
```

Comment:
Why are they 2 addRating fns and what's the purpose of adding a rating without being linked to a UID?

Reply:
This is a vestige of my basic UI debugging to see if the mvvm worked, good call I'm deleting it.